scripts/dts/extract/clocks.py

```diff
@@ -5,30 +5,168 @@
 #
 
 from extract.globals import *
+from extract.edts import *
 from extract.directive import DTDirective
+from extract.default import default
 
 ##
 # @brief Manage clocks related directives.
 #
 # Handles:
 # - clocks
-# directives.
```

Comment:
Limit changes to what is supported in master today.
Once again, advanced clock support in dts is a desirable feature, but as you see integrating a significant feature is a long journey. Splitting the code is key for efficient process.

Reply:
Limited